### PR TITLE
added umichigan

### PR DESCRIPTION
**Pull request by berrakozer**
@sbillinge the other two entries passed the test but they are under separate pull requests. I am making this one as a separate pull request as well but for the rest could you remind me how to pile them under one pull request?

### CODE REVIEW

**Comment by sbillinge** [comment]
Hi @berrakozer to add something to a PR you just push to the branch, so to add to this PR you would
```
git checkout add_umichigan
edit edit edit, work work work, add new institutions, etc.
git add path/to/institutions.yml
git commit -m "adding this and that"
git push  origin add_umichigan
```
The new edits will simply appear on the PR automatically

File: db/institutions.yml [hunk 1]
```diff
@@ -584,6 +606,29 @@ lehighu:
...
   state: PA
   zip: '18015'
+m.p.instituteofsocialscienceresearch:
+  aka:
+    - MPISSR
+  city: Ujjain
+  country: India
+  day: 19
+  month: 2
+  name: M.P. Institute of Social Science Research
```
Comment (sbillinge) [inline db/institutions.yml]: This should be max planck institution for solid state research, or sthg like that.

Comment (berrakozer) [reply db/institutions.yml]: @sbillinge this one is called "M.P. Institute of Social Science Research, Ujjain" on its official website corresponding to "mpissr" acronym. "Max Planck Institute for Chemical Physics of Solids" is "mpicps" that is in Germany.

**Comment by berrakozer** [comment]
@sbillinge all the missing institutions you have gave me are committed on this pull request so PR is ready for your evaluation.

File: db/institutions.yml [hunk 2]
```diff
@@ -629,6 +629,18 @@ maxplanckinstituteforchemicalphysicsofsolids:
   uuid: 15f283f2-0131-47b7-8e4d-4d14db0b973a
   year: 2020
   zip: '01187'
+maxplanckinstituteforsolidstateresearch:
```
Comment (sbillinge) [inline db/institutions.yml]: let's make this mpissr too.

**Pull request merged**